go/internal/indexerhandler/launchpad.go

```diff
@@ -0,0 +1,188 @@
+package indexerhandler
+
+import (
+	"encoding/json"
+
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	"github.com/TERITORI/teritori-dapp/go/internal/indexerdb"
+	"github.com/TERITORI/teritori-dapp/go/pkg/launchpadpb"
+	"github.com/friendsofgo/errors"
+	"go.uber.org/zap"
+)
+
+func (h *Handler) handleExecuteSubmitCollection(e *Message, execMsg *wasmtypes.MsgExecuteContract) error {
+	var jsonData map[string]map[string]interface{}
+	if err := json.Unmarshal(execMsg.Msg.Bytes(), &jsonData); err != nil {

```

Comment:
would be nice to unmarshal into a type ? instead of using jsonData["key"] after isn't ?

Reply:
You're right, I'll do that

Reply:
https://github.com/TERITORI/teritori-dapp/pull/1432/commits/b4ec5b32e099b7955d6075367325d3b21697c9a8#diff-72d2509d3301641260ef684ddb3c1d30b888f8ff23461ec5dbed016789d4cd87R65-R76